test/integration/testExecute.js

```diff
@@ -177,6 +179,9 @@ describe('Execute test - variant', function ()
   const TEST_VARIANT_FORMAT = "TEST_VARIANT_FORMAT";
   const TEST_COL = "COL";
   const TEST_HEADER = "ROOT";
+  const TEST_ATTRIBUTE_NAME = "attr";
```

Comment:
do we need constants for those elements?

Reply:
but why they are constants? isn't that sth that could be written for single test?

Reply:
they could be local for the test - so maybe we don't need them?

Reply:
It is useful for me to have that string like a const and use in given part and assertion checks.